Fix null and empty values in browser CSV export.

oskarhane opened the pull request.

oskarhane committed:
Fix null and empty values.

thobe commented:
Where are the tests that ensures that what we export through the CSV export feature can be re-imported through `LOAD CSV`?

oskarhane committed:
Create CSV file for import tests.

- Refactor Utils and Serializer to stand alone modules.
- Add script to create "/tmp/csv-export-javascript.csv" output file.
- Invoke script when test runs.
